src/index.ts

```diff
@@ -190,6 +217,10 @@ class TaskInfo extends AsyncResource {
     this.runInAsyncScope(this.callback, null, err, result);
     this.emitDestroy(); // `TaskInfo`s are used only once.
   }
+
+  get [kQueueOptions] () : any | null {
```

Comment:
The docs say this is `object | null` – this should either be `object | null`, or if not, then `any`, because `any | null` is the same as `any` :)